docs/development/testguide.rst

```diff
@@ -1077,6 +1077,14 @@ In some cases, you may see failures on continuous integration services that
 you do not see locally, for example because the operating system is different,
 or because the failure happens with only 32-bit Python.
 
+Astropy also has the option to run a comparative benchmark using Github Actions
+to test a new pull request against the current main branch. It uses the benchmarks
+from `Astropy Benchmarks <https://github.com/astropy/astropy-benchmarks/>`_. This
+action only runs if a maintainer adds the label `benchmark` to the pull request.
```

Comment:
```suggestion
action only runs if a maintainer adds the label ``benchmark`` to the pull request.
```

To avoid `docs/development/testguide.rst:1080: WARNING: py:obj reference target not found: benchmark`